joomla/joomla-cms
# [5.2] Usability: Back-end - Add total number of items to Page Navigation

Pull Request for Issue ##43548 .

### Summary of Changes
Adds the total number of items at the end of a table

![image](https://github.com/joomla/joomla-cms/assets/1296369/ee62c33e-b646-4efc-bceb-ff39c9ff0e52)

The total will be present on any list view that uses `<?php echo $this->pagination->getListFooter(); ?>`

### Testing Instructions
Apply PR and check the bottom of the table
Make sure that the total is correct after filters have been applied
Make sure that the total is displayed even when the pagination is not applied


### Link to documentations
Please select:
